lib/propolis/src/block/minder.rs

```diff
@@ -327,6 +327,18 @@ impl QueueMinder {
         }
     }
 
+    /// Add a set of workers to be notified when this queue may have requests
+    /// available.
+    ///
+    /// This should only be called with the remaining parts of a bitmap obtained
+    /// from an ealier `take_notifications`. Using other bit patterns may result
+    /// in wakeups to out-of-range worker IDs and subsequent panic.
+    pub(crate) fn add_notifications(&self, worker_ids: Bitmap) {
```

Comment:
turbo nit: perhaps this ought to be 
```suggestion
    pub(in crate::block) fn add_notifications(&self, worker_ids: Bitmap) {
```
since it has some module-local invariants and we don't expect it to be called elsewhere?